### PR TITLE
Switch to next

### PR DESCRIPTION
**Pull request by kkavod**
Need to do research on why basic Janus tests don't work and then implement proper E2E

### CODE REVIEW

File: frontend/src/app/components/channel/channel.controller.ts [hunk 1]
```diff
@@ -6,53 +6,118 @@ export interface IChannelScope extends ng.IScope {
...
+  next() {
+    if (!this.onlineUsers.length) {
+      this.programUser = this.previewUser = null;
+      return;
+    }
+
+    // Clone the video to program
+    this.$scope.programElement.src = this.$scope.previewElement.src;
+
+    // TODO: trigger switch from Janus here
```
Comment (edoshor) [inline frontend/src/app/components/channel/channel.controller.ts]: Make the janus switch first thing.

What if the janus rest api call fails ? 
We need to communicate it to the user somehow.

File: frontend/src/app/components/channel/channel.controller.ts [hunk 2]
```diff
@@ -6,53 +6,118 @@ export interface IChannelScope extends ng.IScope {
...
-    // this.users[login].joined = moment();
-
-    // Means he sends video/audio to janus
-    // Now we decide to get his video/audio here or not.
-    // If program or preview ==> get stream + show on video element
-    if (!this.previewUserLogin) {
-      this.previewUserLogin = login;
-      var element = this.scope.selfElement.find('.preview').get(0);
-      this.janus.attachRemoteHandle(login, element);
+    // TODO: The timestamp should be better taken from Janus point of view
```
Comment (edoshor) [inline frontend/src/app/components/channel/channel.controller.ts]: Implement all TODOs

File: frontend/src/app/components/channel/channel.html [hunk 3]
```diff
@@ -1,35 +1,58 @@
...
+      </md-grid-tile-header>
+    </md-grid-tile>
+  </md-grid-list>
+
+  <md-list class="users">
+    <!-- Online users -->
+    <md-list-item ng-repeat="user in vm.onlineUsers">
+      <!-- #9E9E9E -->
+
+      <md-icon layout="column" aria-label="" ng-style="{color: '#4CAF50'}" class="material-icons">lens</md-icon>
```
Comment (edoshor) [inline frontend/src/app/components/channel/channel.html]: Why are we using inline styles ?
Shouldn't we be using css ?

File: frontend/src/app/components/channel/channel.controller.ts (hunk 1 above)
Comment (kkavod) [inline frontend/src/app/components/channel/channel.controller.ts]: Kolman's working on it

File: frontend/src/app/components/channel/channel.controller.ts (hunk 2 above)
Comment (kkavod) [inline frontend/src/app/components/channel/channel.controller.ts]: Don't know a straightforward way to do it for now, also current approach works in most cases.

File: frontend/src/app/components/channel/channel.html (hunk 3 above)
Comment (kkavod) [inline frontend/src/app/components/channel/channel.html]: It is from Angular Material examples, but will try to move to scss

Comment (kkavod) [inline frontend/src/app/components/channel/channel.html]: Leaving as is, don't want to focus on thinking about CSS schema now, also as i said this approach is used all over Angular Material examples. Moved video's width style to scss.

**Pull request merged**